src/frontend/citizen-portal/src/app/components/shell-ticket/shell-ticket.component.ts

```diff
@@ -146,15 +157,21 @@ export class ShellTicketComponent implements OnInit {
 
       this.ticketImageSrc = shellTicketData.ticketImage;
       this.ticketFilename = shellTicketData.filename;
-      if (this.appConfigService.useMockServices) {
-        this.progressRef.complete();
-      } else {
-        this.recognizeContent(shellTicketData.ticketFile);
-      }
+      // if (this.appConfigService.useMockServices) {
```

Comment:
do you plan on putting this back? if you dont want to use this code, can it not be removed?